test/cpp/interop/interop_server.cc

```diff
@@ -321,6 +322,9 @@ void grpc::testing::interop::RunServer(
   ServerBuilder builder;
   builder.RegisterService(&service);
   builder.AddListeningPort(server_address.str(), creds);
+  if (FLAGS_max_send_message_size >= 0) {
```

Comment:
https://github.com/grpc/grpc/pull/7846/files#diff-5dd9c06965eebe9bf2d14a72f1c8d064R165 ignores values <= 0, but 0 is accepted here.